src/main/java/hudson/plugins/git/UserMergeOptions.java

```diff
@@ -18,7 +19,7 @@
 public class UserMergeOptions extends AbstractDescribableImpl<UserMergeOptions>  implements Serializable {
 
     private String mergeRemote;
-    private String mergeTarget;
+    private final String mergeTarget;
```

Comment:
Why not use the `MergeCommand.Strategy` type here?
For the fast forward mode (`MergeCommand.GitPluginFastForwardMode`) that's already the case.